### ISSUE TITLE
arxiv-add-bibtex-entry throws error "Invalid function: path/to/.bib"

### ISSUE THREAD

**Issue by albertfgu** [jkitchin/org-ref#1039]
For context: I use Spacemacs and org-roam + org-roam-bibtex, which depends on org-ref. I recently upgraded my packages after about ~2 years, and am running into some issues. I commented out the org-roam-bibtex stuff and am running into the following problem.

----------

When calling `arxiv-add-bibtex-entry`, I get the error
```
helm-M-x-execute-command: Invalid function: "/Users/albertgu/temp.bib"
```

This file name appears nowhere in my config except as the variable `bibtex-completion-bibliography`. This snippet contains all the bibtex-related parts of my config:
```emacs-lisp
(setq bibtex-completion-bibliography (file-truename "~/temp.bib")
         bibtex-completion-library-path "~/Dropbox/org/bibtex/pdf/"
         bibtex-completion-notes-path "~/Dropbox/org/bibtex/helm-bibtex-notes/"
         bibtex-completion-notes-template-multiple-files "* ${author-or-editor}, ${title}, ${journal}, (${year}) :${=type=}: \n\nSee [[cite:&${=key=}]]\n"
   )

   ;; open pdf with system pdf viewer (works on mac)
   (setq bibtex-completion-pdf-open-function
         (lambda (fpath)
           (start-process "open" "*open*" "open" fpath)))

   ;; From example org-ref config https://github.com/jkitchin/org-ref/blob/master/melpa/init-helm.el
   (setq org-ref-insert-link-function 'org-ref-insert-link-hydra/body
         org-ref-insert-cite-function 'org-ref-cite-insert-helm
         org-ref-insert-label-function 'org-ref-insert-label-link
         org-ref-insert-ref-function 'org-ref-insert-ref-link
         org-ref-cite-onclick-function (lambda (_) (org-ref-citation-hydra/body)))

   (define-key org-mode-map (kbd "C-c ]") 'org-ref-insert-link)
```

If I don't set the variable `bibtex-completion-bibliography` then `arxiv-add-bibtex-entry` does work upon manually entering a bib file.

**Comment by jkitchin** [comment]
It could be because bibtex-completion-bibliography was set to a string, and not a list. the push above might fix that for you.

**Comment by albertfgu** [comment]
D'oh, that was it. Incidentally, what's the proper way to turn a string into a list? Should I have replaced `(file-truename "~/temp.bib")` with `(list (file-truename "~/temp.bib"))`? 

Also, I'm now getting the following error in my message buffer:
```
Wrote /Users/albertgu/Dropbox/org/bibtex/ref.bib
Parsing bibliography file /Users/albertgu/Dropbox/org/bibtex/ref.bib ...
Resolving cross-references ...
Done (re)loading bibliography.
doi-utils-async-download-pdf: nil
```

I see something similar discussed in https://github.com/jkitchin/org-ref/issues/923, but it seemed to have been resolved there. Should I try the fix of adding `(setq doi-utils-async-download nil)` anyways?
